src/rhub/api/monitor.py

```diff
@@ -0,0 +1,124 @@
+import os
```

Comment:
Please, move this and the monitoring_*.yml to a directory, to keep them together.

Reply:
Thanks, looking better now. Fixed in the last commit.